util/opentelemetry-util-tracer/src/opentelemetry/util/tracer/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry import trace
+from opentelemetry.trace.span import Span
+from typing import Optional 
+from opentelemetry.util import types
```

Comment:
I didn't know this until I just searched it up, but it seems like these module imports might need to be formatted a specific way (according to [the Python style guide](https://www.python.org/dev/peps/pep-0008/#imports)). Something like this, with an empty line separating the groups? 😄 
```suggestion
from typing import Optional

from opentelemetry import trace
from opentelemetry.trace.span import Span
from opentelemetry.util import types
```